docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+- **Description**: The maximum loading lag that can be tolerated by a BE replica. If this value is exceeded, cloning is performed to clone data from other replicas. Unit: seconds.
+
+##### desired_max_waiting_jobs
+
+- **Unit**: -
+- **Default**: 1024
+- **Description**: The maximum number of pending jobs in an FE. The number refers to all jobs, such as table creation, loading, and schema change jobs. If the number of pending jobs in an FE reaches this value, the FE will reject new load requests. This parameter takes effect only for asynchronous loading. From v2.5 onwards, the default value is changed from 100 to 1024.
+
+##### max_load_timeout_second
+
+- **Unit**: s
+- **Default**: 259200
+- **Description**: The maximum timeout duration allowed for a load job. The load job fails if this limit is exceeded. This limit applies to all types of load jobs. Unit: seconds.
+
+##### min_load_timeout_second
+
+- **Unit**: s
+- **Default**: 1
+- **Description**: The minimum timeout duration allowed for a load job. This limit applies to all types of load jobs. Unit: seconds.
+
+##### max_running_txn_num_per_db
+
+- **Unit**: -
+- **Default**: 100
+- **Description**: The maximum number of load transactions allowed to be running for each database within a StarRocks cluster. The default value is `100`. When the actual number of load transactions running for a database exceeds the value of this parameter, new load requests will not be processed. New requests for synchronous load jobs will be denied, and new requests for asynchronous load jobs will be placed in queue. We do not recommend you increase the value of this parameter because this will increase system load.
+
+##### load_parallel_instance_num
+
+- **Unit**: -
+- **Default**: 1
+- **Description**: The maximum number of concurrent loading instances for each load job on a BE.
+
+##### disable_load_job
+
+- **Unit**: -
+- **Default**: FALSE
+- **Description**: Whether to disable loading when the cluster encounters an error. This prevents any loss caused by cluster errors. The default value is `FALSE`, indicating that loading is not disabled.
+
+##### history_job_keep_max_second
+
+- **Unit**: s
+- **Default**: 604800
+- **Description**: The maximum duration a historical job can be retained, such as schema change jobs, in seconds.
+
+##### label_keep_max_num
+
+- **Unit**: -
+- **Default**: 1000
+- **Description**: The maximum number of load jobs that can be retained within a period of time. If this number is exceeded, the information of historical jobs will be deleted.
+
+##### label_keep_max_second
+
+- **Unit**: s
+- **Default**: 259200
+- **Description**: The maximum duration the labels of load jobs that have been completed and are in the FINISHED or CANCELLED state can be retained in the StarRocks system. The default value is 3 days. After this duration expires, the labels will be deleted. This parameter applies to all types of load jobs. Unit: seconds. A value too large consumes a lot of memory.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/4224bb67-c616-41df-974a-a39d0378b7e8)
I'm not sure if "The default value is 3 days." is clear and won't solicit ambiguity, because the unit is "s" and "the value" can't be "3 days" but a much longer integer measured in seconds. So, I think it's better to say, for example, "The default maximum duration is 3 days."